dGame/dComponents/TriggerComponent.cpp

```diff
@@ -75,8 +100,13 @@ void TriggerComponent::HandleTriggerCommand(LUTriggers::Command* command, Entity
 			case eTriggerCommandType::STOP_PATHING: break;
 			case eTriggerCommandType::START_PATHING: break;
 			case eTriggerCommandType::LOCK_OR_UNLOCK_CONTROLS: break;
-			case eTriggerCommandType::PLAY_EFFECT: break;
-			case eTriggerCommandType::STOP_EFFECT: break;
+			case eTriggerCommandType::PLAY_EFFECT:
+				HandlePlayEffect(targetEntity, argArray);
+				break;
+			case eTriggerCommandType::STOP_EFFECT:
+				GameMessages::SendStopFXEffect(targetEntity, true, command->args);
+				break;
+			// DEPRECATED BLOCK START
```

Comment:
Thoughts on moving this closer to the default so all the unhandled triggers are in one block?